content/en/docs/distributions/aws/component-guides/pipeline.md

```diff
@@ -0,0 +1,171 @@
++++
+title = "Kubeflow Pipelines on AWS"
+description = "Get started with Kubeflow Pipelines on Amazon EKS"
+weight = 20
++++
+
+For more information about Kubeflow Pipelines, see [Connecting to the Kubeflow Pipelines using the SDK client](https://www.kubeflow.org/docs/components/pipelines/sdk/connect-api/). 
+
+## Authenticate Kubeflow Pipelines using SDK inside cluster
+
+Refer to the following guide to connect to Kubeflow Pipelines from [inside your cluster](https://www.kubeflow.org/docs/components/pipelines/sdk/connect-api/#connect-to-kubeflow-pipelines-from-the-same-cluster).
+
+## Authenticate Kubeflow Pipelines using SDK outside cluster
+
+Refer to the following guide to connect to Kubeflow Pipelines from [outside your cluster](https://www.kubeflow.org/docs/components/pipelines/sdk/connect-api/#connect-to-kubeflow-pipelines-from-outside-your-cluster).
+
+Refer to the following steps to use `kfp` to pass a cookie from your browser after you log into Kubeflow. The following example uses a Chrome browser.
+
+> Note: It is necessary to use the latest container images listed in [AWS-Optimized Kubeflow Notebooks](/docs/distributions/aws/notebook-server) because they include a critical [SDK fix](https://github.com/kubeflow/pipelines/pull/4285).
+
+<img src="/docs/images/aws/kfp-sdk-browser-cookie.png"
+  alt="KFP SDK Browser Cookie"
+  class="mt-3 mb-3 border border-info rounded">
+
+<img src="/docs/images/aws/kfp-sdk-browser-cookie-detail.png"
+  alt="KFP SDK Browser Cookie Detail"
+  class="mt-3 mb-3 border border-info rounded">
+
+Once you get a cookie, authenticate `kfp` by passing the cookie from your browser. Use the session based on the appropriate manifest for your deployment, as done in the following examples. 
+
+### **Dex** 
+
+If you want to use port forwarding to access Kubeflow, run the following command and use `http://localhost:8080/pipeline` as the host.
+
+```bash
+kubectl port-forward svc/istio-ingressgateway -n istio-system 8080:80
+```
+
+Pass the cookie from your browser: 
+```bash
+# This is the "Domain" in your cookies. Eg: "localhost:8080" or "<ingress_alb_address>.elb.amazonaws.com"
+kubeflow_gateway_endpoint="<YOUR_KUBEFLOW_GATEWAY_ENDPOINT>"
+
+authservice_session_cookie="<YOUR_COOKIE>"
+
+namespace="<YOUR_NAMESPACE>"
+
+client = kfp.Client(host=f"http://{kubeflow_gateway_endpoint}/pipeline", cookies=f"authservice_session={authservice_session_cookie}")
+client.list_experiments(namespace=namespace)
+```
+
+If you want to set up application load balancing (ALB) with Dex, see [Connect to Your Kubeflow Cluster](https://github.com/awslabs/kubeflow-manifests/tree/v1.3-branch/distributions/aws/examples/vanilla#connect-to-your-kubeflow-cluster) and use the ALB address as the Kubeflow Endpoint. 
```

Comment:
If you want to set up application load balancing (ALB) with Dex, see [Exposing Kubeflow over Load Balancer](https://github.com/awslabs/kubeflow-manifests/tree/v1.3-branch/distributions/aws/examples/vanilla#exposing-kubeflow-over-load-balancer) and use the ALB address as the Kubeflow Endpoint.